android/src/test/java/org/mozilla/mozstumbler/client/UpdaterTest.java

```diff
@@ -59,16 +62,14 @@ public void testUpdater() {
         ServiceLocator.getInstance().putService(IHttpUtil.class, mockHttp);
         ServiceLocator.getInstance().putService(ISystemClock.class, clock);
 
-        // wifi is always unavailable
+        // network is always available
         doReturn(false).when(upd).wifiExclusiveAndUnavailable(Mockito.any(Context.class));
 
         assertFalse(upd.checkForUpdates(activity, ""));
-        now += Updater.UPDATE_CHECK_FREQ_MS;
```

Comment:
I generally like this test, but let's split it up.   The `testUpdater()` method is meant to just test the parsing of the version numbers.  

Let's have one test for the version number parsing, and a separate one to verify that a `checkForUpdates` call that returns false does not update the sLastUpdate timestamp.